lib/org/chromium/webidl/PipelineRunner.js

```diff
@@ -14,7 +14,7 @@ foam.CLASS({
     'org.chromium.webidl.PipelineMessage',
   ],
 
-  imports: [ 'outputDAO?' ],
+  imports: [ 'outputDao? as dao' ],
```

Comment:
Ditto in this class: comments on import, property naming, and property `class`/`of`.